include/libnuraft/raft_server.hxx

```diff
@@ -284,6 +289,11 @@ public:
          * server's term does not match the given term.
          */
         uint64_t expected_term_;
+
+        /**
+         * Opaque cookie which will be passed as is to the req_ext_cb
+         */
+        void* context{nullptr};
```

Comment:
Can you please align the name with the existing ones? (`context_`)

Reply:
oh sorry I thought I did, missed it. Will update it